src/commands/mirror_node.mjs

```diff
@@ -172,7 +178,50 @@ export class MirrorNodeCommand extends BaseCommand {
             rendererOptions: constants.LISTR_DEFAULT_RENDERER_OPTION
           })
         }
-      }
+      },
+      {
+        title: 'Seed DB data',
+        task: async (ctx, parentTask) => {
+          const subTasks = [
+            {
+              title: 'Insert data in public.file_data',
+              task: async (ctx, _) => {
+                const namespace = self.configManager.getFlag(flags.namespace)
+
+                const feesFileIdNum = 111;
+                const exchangeRatesFileIdNum = 112;
+                const timestamp = Date.now();
+
+                const fees = await getFileContents(this.accountManager, namespace, feesFileIdNum);
+                const exchangeRates = await getFileContents(this.accountManager, namespace, exchangeRatesFileIdNum);
+
+                const importFeesQuery = `INSERT INTO public.file_data(file_data, consensus_timestamp, entity_id, transaction_type) VALUES (decode('${fees}', 'hex'), ${timestamp + '000000'}, ${feesFileIdNum}, 17);`
+                const importExchangeRatesQuery = `INSERT INTO public.file_data(file_data, consensus_timestamp, entity_id, transaction_type) VALUES (decode('${exchangeRates}', 'hex'), ${
+                    timestamp + '000001'
+                }, ${exchangeRatesFileIdNum}, 17);`;
+                const sqlPath = path.resolve(constants.TEMP_DIR, 'importFeesAndExchangeRates.sql');
+
+                fs.writeFileSync(  sqlPath, [importFeesQuery, importExchangeRatesQuery].join(`\n`) );
+
+                const scriptName = 'insert-fee-data.sh'
+                const scriptPath = path.join(constants.RESOURCES_DIR, scriptName)
+
+                exec(`sh ${scriptPath}`, (error, stdout, stderr) => {
```

Comment:
Instead of running the bash script, we need to leverage the `K8` class that uses the Kubernetes API: https://github.com/hashgraph/solo/blob/main/src/core/k8.mjs#L606